05_multistep_td.py

```diff
@@ -204,7 +205,7 @@ def train_step(self):
         target_q = self.get_target_value(self.b_next_states)    # get target q-value through the target network
         # get td_targets of batch states
         td_target = self.b_rewards + \
-            self.gamma * target_q[np.arange(target_q.shape[0]), best_action_idxes] * (1 - self.b_dones)
+            self.gamma**(self.n_step ) * target_q[np.arange(target_q.shape[0]), best_action_idxes] * (1 - self.b_dones)
```

Comment:
and this place (btw, it has a unused blank)

Reply:
TD =0 : G = r+ gamma** argmax_{a'}Q(s',a')

n_steps: G = r  + gamma* r(t+1) + gammer^2 r(t+2) + gamma^3 argmax_{a}Q(s_{t+n},a)

In your code self.b_rewards = = r  + gamma* r(t+1) + gammer^2 r(t+2)

Reply:
Ok, I got it, just need to remove the redundant blank in "self.n_step "